File: 002/project_euler_002.c
#include <stdio.h>

int x = y = 1;
int sum = aux = 0;

int main() {
  while (y < 4000000) {
    if (y % 2 == 0) {
      sum += y;
    }
    aux = x;
    x += y;
    y = aux;
  }

  printf("%i", sum);

  return 0;
}

File: 002/project_euler_002.java
public class project_euler_002 {
  public static void main(String [] args) {
    int x = y = 1;
    int sum = aux = 0;

    while (i < 4000000) {
      if (x % 2 == 0) {
        sum += x;
      }
      aux = x;
      x += y;
      y = aux;
    }

    System.out.println(sum);
  }
}
